packages/solid/store/src/store.ts

```diff
@@ -219,17 +231,14 @@ export function updatePath(current: StoreNode, path: any[], traversed: PropertyK
   } else setProperty(current, part, value);
 }
 
-export type DeepReadonly<T> = 0 extends 1 & T
```

Comment:
Probably this was necessary in an older version, but isn't anymore (at least not in my tests).

Reply:
Thanks